docs/operator-manual/server-commands/argocd-application-controller.md

```diff
@@ -0,0 +1,52 @@
+## argocd-application-controller
+
+application-controller is a controller to operate on applications CRD
+
+### Synopsis
+
+application-controller is a controller to operate on applications CRD
```

Comment:
Same comment as 'argocd-server' below, maybe something like:

"This command starts the application controller, which is a Kubernetes controller that continuously monitors running applications and compares the current, live state against the desired target state (as specified in the repo). This server may be customized via the command parameters below."

Reply:
Updated.